src/plugin.js

```diff
@@ -523,15 +522,16 @@ class VR extends Plugin {
           // Native WebVR Head Mounted Displays (HMDs) like the HTC Vive
           // also need the cardboard button to enter fully immersive mode
           // so, we want to add the button if we're not polyfilled.
+
+          this.controls3d = new VRControls(this.camera);
```

Comment:
should this line be moved above the comment so the comment is attached to the if statement below?